_posts/2016-05-10-finite-state-machines-with-akka.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+name: Finite-state-machines-with-Akka
+title: 'Finite state machines with Akka'
+date: 2016-05-10 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/machine.png
+    attribution:
+      text: Machine by Olle Svensson  
+      href: https://www.flickr.com/photos/ollesvensson/3335131082
+tags:
+- akka
+- state
+- fsm
```

Comment:
did you check with the thesaurus and word cloud generator, more interesting words will come out.